src/main/scala/co/teapot/tempest/graph/MemoryMappedDirectedGraphConverter.scala

```diff
@@ -180,7 +179,7 @@ class MemoryMappedDirectedGraphConverter(val edgeListFile: File,
 
         // Store segment size
         val edgeCount = (distinctNeighborArrayLists map { _.size.toLong }).sum
-        if (dir == EdgeDir.Out)
+        if (dir == EdgeDirOut)
```

Comment:
Good question.  New comment:
`if (dir == EdgeDirOut) // avoid double counting both out and in edges`